roles/ssh_hardening/tasks/hardening.yml

```diff
@@ -50,6 +50,26 @@
   ansible.builtin.include_tasks: crypto_kex.yml
   when: not ssh_kex
 
+- name: "Find host public key files"
+  ansible.builtin.find:
+    depth: 1
+    file_type: file
+    paths:
+      - "{{ ssh_host_keys_dir }}"
+    patterns:
+      - "ssh_host_*.pub"
+  when: ssh_server_hardening | bool
+  register: ssh_host_pub_keys
+
+- name: "Change host public key ownership, group and permissions"
+  ansible.builtin.file:
+    path: "{{ item.path }}"
+    owner: "{{ ssh_host_keys_owner }}"
+    group: "{{ ssh_host_keys_group }}"
+    mode: "{{ ssh_host_pub_keys_mode }}"
+  when: ssh_server_hardening | bool
+  loop: "{{ ssh_host_pub_keys.files }}"
+
```

Comment:
I'd prefer these tasks to be in `crypto_hostkeys.yml`

Reply:
@schurzi I was thinking the same, but then this will only get executed if the user provides their own `ssh_host_key_files`:
https://github.com/dev-sec/ansible-collection-hardening/blob/master/roles/ssh_hardening/tasks/hardening.yml#L35-L39

Should I maybe move the keypair creation conditional into a block inside `crypto_hostkeys.yml`?